--- origsrc/ibus-1.4.2/bindings/vala/Makefile.am	2012-03-06 09:34:36.000000000 -0600
+++ src/ibus-1.4.2/bindings/vala/Makefile.am	2012-11-09 00:17:41.528222000 -0600
@@ -31,14 +31,14 @@ vapi_deps = \
 	$(srcdir)/IBus-1.0.metadata \
 	$(srcdir)/IBus-1.0-custom.vala \
 	| \
-	$(top_srcdir)/src/IBus-@IBUS_API_VERSION@.gir \
+	$(top_builddir)/src/IBus-@IBUS_API_VERSION@.gir \
 	$(NULL)
 
 ibus-@IBUS_API_VERSION@.vapi: $(vapi_deps)
 	$(AM_V_GEN) $(VAPIGEN) --library ibus-@IBUS_API_VERSION@ \
 			--pkg gio-2.0 \
 			--metadatadir=$(srcdir) \
-			$(top_srcdir)/src/IBus-@IBUS_API_VERSION@.gir \
+			$(top_builddir)/src/IBus-@IBUS_API_VERSION@.gir \
 			$(srcdir)/IBus-1.0-custom.vala
 
 EXTRA_DIST = \
--- origsrc/ibus-1.4.0/src/Makefile.am	2011-09-17 11:14:51.000000000 -0500
+++ src/ibus-1.4.0/src/Makefile.am	2011-10-02 12:59:24.402445000 -0500
@@ -174,7 +174,7 @@ IBus_1_0_gir_SCANNERFLAGS = --pkg=glib-2
 IBus_1_0_gir_INCLUDES = GLib-2.0 GObject-2.0 Gio-2.0
 IBus_1_0_gir_LIBS = $(libibus)
 IBus_1_0_gir_FILES = $(addprefix $(srcdir)/,$(introspection_files))
-IBus_1_0_gir_CFLAGS = -DIBUS_COMPILATION
+IBus_1_0_gir_CFLAGS = -DIBUS_COMPILATION -I$(srcdir)
 INTROSPECTION_GIRS += IBus-1.0.gir
 
 girdir = $(datadir)/gir-1.0
