--- origsrc/ibus-1.4.0/ibus/common.py	2011-08-14 18:14:42.000000000 -0500
+++ src/ibus-1.4.0/ibus/common.py	2011-10-02 13:33:57.045637300 -0500
@@ -58,7 +58,7 @@ __all__ = (
 import os
 import sys
 from xdg import BaseDirectory
-import ctypes
+import ctypes, ctypes.util
 
 # __display = os.environ["DISPLAY"]
 # __hostname, __display_screen = __display.split(":", 1)
@@ -104,7 +104,7 @@ import ctypes
 #         return None
 #     return address
 
-libibus = ctypes.CDLL("libibus-1.0.so.0")
+libibus = ctypes.CDLL(ctypes.util.find_library("ibus-1.0"))
 get_address = libibus.ibus_get_address
 get_address.restype=ctypes.c_char_p
 
