--- origsrc/ibus-1.4.0/client/gtk2/Makefile.am	2011-08-10 19:49:42.000000000 -0500
+++ src/ibus-1.4.0/client/gtk2/Makefile.am	2011-10-02 13:13:40.769689800 -0500
@@ -52,6 +52,7 @@ im_ibus_la_LIBADD = \
 im_ibus_la_LDFLAGS = \
 	-avoid-version \
 	-module \
+	-no-undefined \
 	$(NULL)
 
 $(libibus):
--- origsrc/ibus-1.4.0/client/gtk3/Makefile.am	2011-08-10 19:49:42.000000000 -0500
+++ src/ibus-1.4.0/client/gtk3/Makefile.am	2011-10-02 13:13:58.781977100 -0500
@@ -52,6 +52,7 @@ im_ibus_la_LIBADD = \
 im_ibus_la_LDFLAGS = \
 	-avoid-version \
 	-module \
+	-no-undefined \
 	$(NULL)
 
 $(libibus):
--- origsrc/ibus-1.5.15/ui/gtk3/Makefile.am	2017-03-13 03:07:58.221338900 -0500
+++ src/ibus-1.5.15/ui/gtk3/Makefile.am	2017-03-13 03:29:38.908252500 -0500
@@ -171,6 +171,9 @@ libibus_emoji_dialog_1_0_la_LDFLAGS =
     -export-symbols-regex "ibus_.*"     \
     -version-info @LT_VERSION_INFO@     \
     $(NULL)
+libibus_emoji_dialog_1_0_la_LIBADD =    \
+    $(libibus)                          \
+    $(GTK3_LIBS)
 libibus_emoji_dialog_1_0_la_SOURCES =   \
     emojier.c                           \
     iconwidget.c                        \
